### ISSUE TITLE
Crashing with optifine?

### ISSUE THREAD

**Issue by SpiritVII** [Theta-Dev/ConstructionWand#31]
Been crashing with optifine. Hope these things help you troubleshoot but I honestly have no idea. Ill let you know if i find anything.

MC version: 1.16.4

Mods:
Forge 1.16.4-forge35.1.37
OptiFine_1.16.4_HD_U_G6_pre10 (and tried with OptiFine_1.16.4_HD_U_G5)
constructionwand-1.16.2-1.7

Crash report:
[crash-2021-01-15_17.49.44-server.txt](https://github.com/Theta-Dev/ConstructionWand/files/5823271/crash-2021-01-15_17.49.44-server.txt)

Other things:
- loads into the main menu fine
- crashes at "100% loading" when loading the world (old or new)
- mod works fine without optifine in the mods folder

**Comment by Theta-Dev** [comment]
Hi, SpiritVII!

First I installed Optifine G5 into my default ConstructionWand testing environment, which currently runs on Forge 35.0.2. This worked without any problems.
![Screenshot_20210116_121237](https://user-images.githubusercontent.com/19636565/104810428-c1345080-57f4-11eb-8e9f-bc8fa56a8831.png)

With the latest Forge version (35.1.37) however, I experienced a similar crash either on startup or upon world creation/loading.
However this kept happening when I took ConstructionWand out and ran MC with only OptiFine G5.

Then I downloaded the most recent version of OptiFine, which is G6_pre12. It loaded and ran fine - both on its own and with ConstructionWand included.

![image](https://user-images.githubusercontent.com/19636565/104810702-bf6b8c80-57f6-11eb-8c48-e965b3a690ca.png)

Concluding, this seems to be a Forge/Optifine compatibility problem and has nothing to do with my mod. Either downgrade Forge or upgrade Optifine to solve the problem.

I hope that I could help you. Keep having fun with my mod!
ThetaDev
